env.yml

```diff
@@ -42,40 +49,37 @@ dependencies:
   - watchfiles
   - pyviz::nbsite # https://panel.holoviz.org/how_to/wasm/sphinx.html
   - jupyter_bokeh # to make panel work in vscode
-  - pip:
-      - qgridnext
-
-  # --- Packages not required for conda-forge recipe ---
-
-  # tardis-sn/nuclear dependencies
-  - beautifulsoup4
-  - lxml
-
+  # - pip:
+  #     - qgridnext
+  
   # Documentation
   - sphinx
   - sphinx-astropy
   - sphinx_bootstrap_theme
+  - sphinx-copybutton
```

Comment:
move to conf.py?

Reply:
Done in https://github.com/tardis-sn/tardis/pull/3063/commits/572b4a3c710c3c4da234ff3cb31db7e165a48a6d